zigbee2mqtt/DOCS.md

```diff
@@ -1,3 +1,34 @@
+# Installation / Configuration after 1.25.0-1
```

Comment:
What about adding these notes to the release notes instead?

Reply:
Have the problem been resolved in the later releases?

Reply:
You mean an automatic migration? Not sure how we can do that.